src/serviceWorker.js

```diff
@@ -14,7 +14,7 @@ const isLocalhost = Boolean(
   window.location.hostname === 'localhost' ||
     // [::1] is the IPv6 localhost address.
     window.location.hostname === '[::1]' ||
-    // 127.0.0.1/8 is considered localhost for IPv4.
+    // 127.0.0.0/8 are considered localhost for IPv4.
```

Comment:
Would you mind reverting this change? Grammatically this should be "is"

Reply:
Ok, done.